Libraries/Text/Text.js

```diff
@@ -228,6 +228,14 @@ const Text: React.AbstractComponent<
 
   const _hasOnPressOrOnLongPress =
     props.onPress != null || props.onLongPress != null;
+  let numberOfLinesKey = 'numberOfLines';
```

Comment:
Why do we need to do this? Can we do it in the native implementation of the fabric view instead of having all these conditions in js?

Reply:
In Fabric Most of layout logic and props is shared between TextComponent and TextInputComponent on the native side. I didn't wanted to split them and `numberOfLines` for some reason works differently on Text and TextInput.
For Text is actually sets maximum  height while in TextInput it's exact height. I can probably change it on paper architecture to use maximumNumberOfLines on the native side. Let me know if that would be cleaner. :D

Reply:
Ok, done it this way